src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+//Solution below slightly adapted from https://github.com/Wincenttjoi/CS2103T-duke-chatbot/blob/master/src/main/java/duke/Duke.java
+
+import duke.*;
+
 public class Duke {
```

Comment:
You could also add some comments here! I think this would allow you to communicate your code effectively!